ang/crmMailing/Recipients.js

```diff
@@ -233,6 +233,9 @@
               if('civicrm_mailing' === rcpAjaxState.entity) {
                 params["api.MailingRecipients.getcount"] = {};
               }
+              else if ('civicrm_group' === rcpAjaxState.entity) {
+                params.extra = {0: "saved_search_id"};
```

Comment:
I think this should be 
```suggestion
                params.extra = ["saved_search_id"];
```